submariner/ipsec-cert-issuer.md

```diff
@@ -0,0 +1,114 @@
+# IPSec Certificate Issuer
+
+## Summary
+
+Submariners current security arrangements, namely using an identical pre-shared key in
+all deployments will fail security assessment in most environments which have to comply
+with a pre-existing security policy and rules on using VPNs in general and IPSEC in particular.
+
+In order to comply the minimum enhancement that needs to be implemented is the use
+of X509 certificates per Gateway. This will require the introduction of a Certificate Authority
```

Comment:
I most likely missed it, but am unclear on how would you establish trust chain with the CA? Its certificate bundle should be present on each GW so it needs to be sync'ed.

Reply:
The CAs cert can be synced to the clusters in the same way the self signed broker cert is synced today (via a volume mounted secret). On GW startup this cert could be read from the volume and then loaded into /etc/pki/tls/certs/ca-bundle.crt
I will add more details on this